### PR TITLE
Enhance limiter section

### PR DESCRIPTION
**Pull request by hjianbo**

### CODE REVIEW

**Comment by hjianbo** [comment]
Do I need to manually pick these changes into the release-51 branch?

File: en_US/configuration/limiter.md [hunk 1]
```diff
@@ -6,23 +6,23 @@ For the moment, you can restrict the message rates from the following perspectiv
 
 | **Type**        | Dashboard UI            | **Description**                           | **Recovery Behavior**           |
 | --------------- | ----------------------- | ----------------------------------------- | ------------------------------- |
-| `bytes_rate`    | Data Publish Rate per Client       | Incoming message size in bytes per second | Pause receiving client messages |
-| `messages_rate`  | Messages Publish Rate per Client   | Incoming messages per second              | Pause receiving client messages |
-| `max_conn_rate` | Maximum Connection Rate per Listener| Connections per second                    | Pause receiving new connections |
+| `bytes_rate`    | Data Publish Rate per Client       | Incoming message size in bytes per second per client | Pause receiving client messages |
+| `messages_rate`  | Messages Publish Rate per Client   | Incoming messages per second  per client | Pause receiving client messages |
+| `max_conn_rate` | Maximum Connection Rate per Listener | Connections per second per client | Pause receiving new connections |
```
Comment (hjianbo) [inline en_US/configuration/limiter.md]:
```suggestion
| `max_conn_rate` | Maximum Connection Rate per Listener | Connections per second per listener | Pause receiving new connections |
```

File: en_US/configuration/limiter.md [hunk 2]
```diff
@@ -6,23 +6,23 @@ For the moment, you can restrict the message rates from the following perspectiv
...
 | --------------- | ----------------------- | ----------------------------------------- | ------------------------------- |
-| `bytes_rate`    | Data Publish Rate per Client       | Incoming message size in bytes per second | Pause receiving client messages |
-| `messages_rate`  | Messages Publish Rate per Client   | Incoming messages per second              | Pause receiving client messages |
-| `max_conn_rate` | Maximum Connection Rate per Listener| Connections per second                    | Pause receiving new connections |
+| `bytes_rate`    | Data Publish Rate per Client       | Incoming message size in bytes per second per client | Pause receiving client messages |
+| `messages_rate`  | Messages Publish Rate per Client   | Incoming messages per second  per client | Pause receiving client messages |
+| `max_conn_rate` | Maximum Connection Rate per Listener | Connections per second per listener | Pause receiving new connections |
 
-Limiter can work on the listener level, for example, to set a Limiter for the default TCP listener, you can work with the code below:
+For example, to set a Limiter for the default TCP listener, you can work with the code below:
```
Comment (Meggielqk) [inline en_US/configuration/limiter.md]:
```suggestion
For example, to set a limiter for the default TCP listener, you can work with the code below:
```

File: en_US/rate-limit/rate-limit.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ EMQX uses the following types of limiters to specify the rate limits:
 | messages_rate | Incoming messages per second per client             | Pause receiving client messages |
 | max_conn_rate | Connections per second per listener                     | Pause receiving new connections |
 
-Limiter can work on the listener level. For example, to set a Limiter for the default TCP listener, You can configure it in emqx.conf as follows:
+For example, to set a Limiter for the default TCP listener, You can configure it in emqx.conf as follows:
```
Comment (Meggielqk) [inline en_US/rate-limit/rate-limit.md]:
```suggestion
For example, to set a limiter for the default TCP listener, you can configure it in emqx.conf as follows:
```

**Review by wivwiv — APPROVED** [review]

**Pull request merged**